backend/src/main/java/gov/cdc/usds/simplereport/api/Translators.java

```diff
@@ -375,4 +375,26 @@ public static String parseOrganizationType(String t) {
       return Optional.of(str);
     }
   }
+
+  public static TestResult convertLoincToResult(String loinc) {
+    switch (loinc) {
+      case "260373001":
```

Comment:
This is a first-pass version of the translation between LOINC and display text; will be addressed further in https://github.com/CDCgov/prime-simplereport/issues/3432

Reply:
where do these LOINCs come from and what do they mean?

Reply:
They come from the LIVD table, and correspond to the codes for "positive", "negative", and invalid sample/undetermined. In the future I'd like to see us using the ReportStream LIVD API to get these values per-device, but for now I think hardcoding them is fine since the LOINCs are the same across all devices.

Reply:
My only concern is that the LIVD table changes all the time, we would like to keep a record of what it meant LOINC meant when it was used

Reply:
Hmm I see what you mean. What would you suggest we do in that case though? Keep a stringified results column on the results table to store what the TestResult was at the time of record?

Reply:
Yeah I wonder if we keep storing the result as positive/negative and we only translate to the LOINC when we need it. i am alright with also storing the stringified version in the test result

Reply:
PR to resolve this is #3555